game-core/src/main/java/games/strategy/triplea/ai/pro/data/ProTerritory.java

```diff
@@ -102,7 +102,7 @@ public ProTerritory(final Territory territory) {
     cantMoveUnits = new ArrayList<>(patd.getCantMoveUnits());
     maxEnemyUnits = new ArrayList<>(patd.getMaxEnemyUnits());
     maxEnemyBombardUnits = new HashSet<>(patd.getMaxEnemyBombardUnits());
-    minBattleResult = patd.getMaxBattleResult();
+    maxBattleResult = patd.getMaxBattleResult();
```

Comment:
@ron-murhammer Any chance this could modify existing behaviour in a bad way now that `maxBattleResult` is initialized with non-0?

Reply:
This is a good fix. This has very minimal impact as this copy method is only used in a very limited manner but no concerns here.

Reply:
@RoiEXLab Good point, I looked into it, and the current calls to `getMaxBattleResult()` do not check for `null` AFAICT from IntelliJ's analysis, so it should be OK for the current code.